docs/development/best-practices.md

```diff
@@ -68,12 +68,14 @@ Function expressions may only be used if dynamically rebinding `this` is needed.
 
 Source: [Google TypeScript Style Guide, function declarations](https://google.github.io/styleguide/tsguide.html#function-declarations).
 
-## Code simplicity
+## Code style
 
-### Write simple code
+### Write understandable code
 
-Simple code is easy to read, review and maintain.
-Choose to write verbose and understandable code instead of "clever" code which might take someone a few attempts to understand what it does.
+Write code that is easier to read, review and maintain.
+Avoid "clever" code that's hard to understand.
+
+Prefer verbose code over optimized hard to understand code.
```

Comment:
A code example would help to understand the difference between "simple code" and "clever code". Do you have a suggestion?

```typescript
// Insert optimized but hard to understand code here

// Insert good code here
```